kuznyechik/src/soft/mod.rs

```diff
@@ -81,7 +81,7 @@ impl AlgorithmName for Kuznyechik {
 #[cfg_attr(docsrs, doc(cfg(feature = "zeroize")))]
 impl Drop for Kuznyechik {
     fn drop(&mut self) {
-        self.keys.zeroize();
+        self.keys.iter_mut().for_each(|key| key.zeroize());
```

Comment:
Unfortunately the `Zeroize` impl on `core::slice::IterMut` couldn't handle this case, I believe because this explicit version works due to implicit deref coercion.